### ISSUE TITLE
[expo-store-review]: Unable to open Google Play In-App Reviews

### ISSUE THREAD

**Issue by Yamini0** [expo/expo#21243]
### Minimal reproducible example

``` 
"expo-store-review": "~6.0.0"
"expo": "^47.0.6"
"expo-linking": "~3.3.0"
"react": "18.1.0"

[
        "expo-build-properties",
        {
          "android": {
            "compileSdkVersion": 33,
            "targetSdkVersion": 33,
            "buildToolsVersion": "33.0.0"
          },
          "ios": {
            "deploymentTarget": "13.0"
          }
        }
      ]
```

### What platform(s) does this occur on?

Android

### Summary

The In-app review popup is not visible on **android**. (FYI: checked for all versions of android), but it is working fine for IOS

### Environment

```
System:
OS: macOS 13.1
Shell: 5.8.1 - /bin/zsh
Binaries:
Node: 19.2.0 - /opt/homebrew/bin/node
Yarn: 1.22.19 - /opt/homebrew/bin/yarn
npm: 8.19.3 - /opt/homebrew/bin/npm
Managers:
CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
SDKs:
iOS SDK:
Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
IDEs:
Android Studio: 2021.3 AI-213.7172.25.2113.9123335
Xcode: 14.2/14C18 - /usr/bin/xcodebuild
npmPackages:
@expo/webpack-config: ^0.17.2 => 0.17.4
expo: ^47.0.6 => 47.0.13
react: 18.1.0 => 18.1.0
react-dom: 18.1.0 => 18.1.0
react-native: 0.70.5 => 0.70.5
react-native-web: ~0.18.7 => 0.18.12
npmGlobalPackages:
expo-cli: 6.1.0
Expo Workflow: managed
```

```
import * as StoreReview from "expo-store-review"


  const openReview = async () => {
    if (await StoreReview.hasAction()) {
      StoreReview.requestReview();
    }
  };
  ```

**Comment by leechoohyoung** [comment]
same here.

**Comment by Luke-lsm** [comment]
☝️

**Comment by mkny** [comment]
Same here. It stopped working recently.

**Comment by pinktonio** [comment]
also not working for me

**Comment by AJGeel** [comment]
Same here. 

Checked various devices (physical and emulators). Works on iOS, but does not work on Android. 
We use these versions:

```
"expo": "^50.0.0",
"expo-store-review": "~6.8.3"
// Others: recommended expo-doctor versions
```

As others seem to have the same issue, I think this issue should be re-opened.

**Comment by rafalnawojczyk** [comment]
Same issue on my end. On Simulator I'm getting this response after awaiting 
`await StoreReview.requestReview();`

response:
` [Error: Android ReviewManager task was not successful]`

**Comment by zegenerative** [comment]
Same here 
* SDK 51.0.31
* Android 14
`[Error: Android ReviewManager task was not successful]`

**Comment by g-hamilton** [comment]
Same here
`[Error: Android ReviewManager task was not successful]`

**Comment by zegenerative** [comment]
Make sure you are running an android simulator that has google play store installed

**Comment by g-hamilton** [comment]
> Make sure you are running an android simulator that has google play store installed

Thanks @zegenerative I just tried this using a Pixel 7 on API 35 with Play Store installed. Seems that the error has gone, and both `StoreReview.isAvailableAsync()` and `StoreReview.hasAction()` are returning `true` - but calling `StoreReview.requestReview()` seems to do nothing now. I checked a standalone Expo preview build and had the same result - no error, but no rating popup like I get on iOS.
